shared/oae/bundles/ui/default.properties

```diff
@@ -498,6 +498,9 @@ USERNAME = Username
 USERNAME_COLON = Username:
 USER_MANAGEMENT = User management
 VIDEO = Video
+VISIBILE_ONLY_TO_SPECIFIED_USERS = Visible only to specified users
```

Comment:
Note that we have to make sure that the tenant displayName replacement in the `LOGGEDIN` descriptions should use the entity tenant and not the current user's tenant